### ISSUE TITLE
structure-tables-key and other options in drush.yml breaks the help for sql-dump

### ISSUE THREAD

**Issue by singularo** [drush-ops/drush#5189]
**Describe the bug**
If you have the following keys in the `drush.yml` then `drush help sql-dump` and the other various combinations of that command die with an error:
```
$ drush help sql:dump
  An option named "structure-tables-key" already exists.
```

Testing and it occurs with any of these values in the `drush.yml`

```
skip-tables-key
structure-tables-key
tables-key
skip-tables-list
structure-tables-list
tables-list
```

**To Reproduce**
Removing the value from the `drush.yml` allows the command to work fine. Putting it back in stops the help working.

**Expected behavior**
Expected to see helpful command line switches for drush sql-dump

**Actual behavior**
Error 'An option named "structure-tables-key" already exists.'

**Workaround**
Commented the line in the `drush.yml` got the help, uncommented the line again so the structure tables worked as expected.

### System Configuration
```
| Q               | A
| --------------- | ---
| Drush version?  | 11.1.1
| Drupal version? | 9.4.3
| PHP version     | 8.0.20
| OS?             | Linux
```

**Additional information**

**Comment by pfrenssen** [comment]
I can confirm this. The error originates in `consolidation/annotated-command` version 4.6.0 . It is solved when updating to version 4.6.1 or higher.

The MR that fixes it is https://github.com/consolidation/annotated-command/pull/275

**Comment by pfrenssen** [comment]
This has already been fixed in Drush 11.4.0.
